deployments/examples/authz/k8s-api-access/create-user.sh

```diff
@@ -16,15 +16,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-USERS=("admin admin" "sue group-a" "bob group-a" "kim group-b" "yono group-b" "anonymous anonymous")
+USERS=("admin" "sue" "bob" "kim" "yono" "anonymous")
+USERS_GROUP=("admin" "group-a" "group-a" "group-b" "group-b" "anonymous")
```

Comment:
Add a comment that the user and group should contain the same number of elements.
If anyone wants no group or no user can we use an empty string, i.e. "" ?